src/soep_preparation/clean_modules/hwealth.py

```diff
@@ -60,42 +60,51 @@ def clean(raw_data: pd.DataFrame) -> pd.DataFrame:
     out["hh_net_overall_wealth_d"] = apply_smallest_float_dtype(raw_data["w011hd"])
     out["hh_net_overall_wealth_e"] = apply_smallest_float_dtype(raw_data["w011he"])
 
-    out["hh_vehicles_value_a"] = apply_smallest_float_dtype(raw_data["v010ha"])
-    out["hh_vehicles_value_b"] = apply_smallest_float_dtype(raw_data["v010hb"])
-    out["hh_vehicles_value_c"] = apply_smallest_float_dtype(raw_data["v010hc"])
-    out["hh_vehicles_value_d"] = apply_smallest_float_dtype(raw_data["v010hd"])
-    out["hh_vehicles_value_e"] = apply_smallest_float_dtype(raw_data["v010he"])
+    out["hh_vehicles_value_a"] = apply_smallest_float_dtype(
```

Comment:
```suggestion
    out["hh_vehicles_value_a"] = fix_missings_make_float(raw_data["v010ha"])
```

Smallest int is fine; float should always be the default 64bit.